examples/drivers/windows/otLwf/command.c

```diff
@@ -127,6 +134,12 @@ otLwfCmdInitialize(
             break;
         }
 
+        NtStatus = otLwfCmdResetDevice(pFilter, FALSE);
+        if (!NT_SUCCESS(NtStatus))
+        {
+            Status = NDIS_STATUS_FAILURE;
+        }
+
     } while (FALSE);
```

Comment:
This is a very common usage in Windows programming. One big benefit over `goto` statements is that you don't have to define all local variables at the beginning of the scope. New variables can be defined as needed. You don't need to worry about warnings related to `goto` skipping variable declaration.